### ISSUE TITLE
[BUG] in Svelte template, sub-components try to load the client before it's ready

### ISSUE THREAD

**Issue by pdaoust** [holochain/scaffolding#424]
## Describe the bug

When you `import` a sub-component into `App.svelte`, it tries to get the app client from the `clientContext` before it's been connected to the conductor. This causes the client to be `undefined` in the sub-component.

## Expected behavior

When the sub-component loads, it shouldn't try to do anything with the client until it's connected to the conductor.

## System information

-   OS: all
-   Browser: all
-   Scaffolding and Nix Version: `hc-scaffold 0.3000.10`

## Steps to reproduce

1. Scaffold a Svelte web hApp with at least one collection.
2. add `import AllFoos from "./dna/zome/AllFoos"` near the top of `ui/src/App.svelte`.
3. Run the hApp with `npm start`

## Screenshots

Sample from the getting started tutorial:

![Image](https://github.com/user-attachments/assets/6e9b384f-c052-40ea-ab36-f0b7a0d27e50)

## Additional context

The race-free solution would be to modify every component template (except App.svelte) to read like this:

```javascript
//...
let client: AppClient | undefined;

//...
onMount(async () => {
  client = (getContext(clientContext) as any).getClient();
```

**Comment by mattyg** [comment]
This was just resolved by #425 right?

**Comment by matthme** [comment]
Yes correct, let's close it :)
